File: app/src/main/java/com/example/mz/udacitypopularmovies/SettingsFragment.java
package com.example.mz.udacitypopularmovies;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

/**
 * Created by mateusz.zak on 27.03.2017.
 */

public class SettingsFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.pref_popmovies);
    }
}

File: app/src/main/java/com/mz/popmovies/SettingsFragment.kt
package com.mz.popmovies

import androidx.preference.PreferenceFragmentCompat
import android.os.Bundle
import com.mz.popmovies.R

/**
 * Created by mateusz.zak on 27.03.2017.
 */
class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.pref_popmovies, rootKey)
    }
}